msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md

```diff
@@ -1,16 +1,16 @@
 ---
```

Comment:
Same comments as given for invoking-task-modules.

Reply:
Awaiting inputs from Wajeed